src/formatDate.js

```diff
@@ -50,7 +50,35 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldForm = [...fromFormat];
+  const oldDate = date.split(fromFormat[3]);
+  const newDate = [];
+  let century = '';
+
+  let years = oldDate[oldForm.indexOf('YY')]
+  || oldDate[oldForm.indexOf('YYYY')];
+  const indexYears = oldDate.indexOf(years);
+  const newYears = toFormat[toFormat.indexOf('YYYY')]
+  || toFormat[toFormat.indexOf('YY')];
+
+  if (newYears.length < years.length) {
+    years = years.slice(-2);
+  } else if (newYears.length > years.length) {
+    if (+years < 30) {
+      century = '20';
+    } else if (+years >= 30) {
+      century = '19';
+    }
```

Comment:
Use ternary operator instead of nesting `if`
```suggestion
    century = +years < 30 ? '20' : '19;
```